File: ShortestWord.js
// Simple, given a string of words, return the length of the shortest word(s).

// String will never be empty and you do not need to account for different data types.



function findShort(s){
    let split = s.split(" ")
    let answer = Infinity
    for (let i = 0; i < split.length; i++){
      if(split[i].length < answer){
        answer = split[i].length
      }
    }
    
    return answer
  }


  /* 
  it is pretty simple. we just just need to check on the way down if our current spot in our split length is smaller than 
  the previous answer. the set answer. could also sort the split array and pop off the first and take the length as well.
  */

File: shortestWord.py
# Simple, given a string of words, return the length of the shortest word(s).

# String will never be empty and you do not need to account for different data types.

def find_short(s):
    splitty = s.split()
    answer = splitty[0]
    for word in splitty:
        if len(word) < len(answer):
            answer = word
    return len(answer)

    
# for this one we need to split the word and set our answer to he first word. then
# we can itterate over the list and replace the answe with one that is shorter. then we
# return the length of the answe.